providers/google/src/airflow/providers/google/cloud/hooks/managed_kafka.py

```diff
@@ -27,15 +31,68 @@
 from airflow.providers.google.common.consts import CLIENT_INFO
 from airflow.providers.google.common.hooks.base_google import GoogleBaseHook
 from google.api_core.gapic_v1.method import DEFAULT, _MethodDefault
-from google.cloud.managedkafka_v1 import Cluster, ManagedKafkaClient, Topic, types
+from google.auth.transport import requests as google_requests
+from google.cloud.managedkafka_v1 import Cluster, ConsumerGroup, ManagedKafkaClient, Topic, types
 
 if TYPE_CHECKING:
+    import google.auth.credentials
```

Comment:
how about importing the only `Credentials` from this path? I only see that we are using
`google.auth.credentials.Credentials` in line 54.